docs/project-structure/genezio-configuration-file.md

```diff
@@ -211,6 +215,23 @@ backend:
         packageManager: pnpm
 ```
 
+### How to set the architecture type
+
+To set the  architecture type for your deployed project, you need to add the `backend.language.architecture` field to the `genezio.yaml` file.
```

Comment:
Well, this is only for the `genezio` cloud provider. I need it to be able to set the lambda architecture to x86 - this will match the github action to the cloud architecture and I will avoid some dependencies nightmare.

Any other cloud providers are in alpha/beta for us and also, they will simply disregard this option. Probably in the long run this is not great UX.